src/audio/data_blob.c

```diff
@@ -185,7 +185,7 @@ int comp_data_blob_set(struct comp_data_blob_handler *blob_handler,
 #endif
 
 	comp_dbg(blob_handler->dev, "comp_data_blob_set_cmd() pos = %d, fragment size = %d",
-		 pos, fragment_size);
+		 pos, (uint32_t)fragment_size);
```

Comment:
how about just using `%zd` or `%zu`? Same everywhere for `ssize_t` or `size_t` printing respectively

Reply:
Done. Went for `%zu` as `ssize_t` doesn't seem to be used anywhere in the places that the compiler raises warnings for.